runelite-client/src/main/java/net/runelite/client/plugins/StatusBars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,301 @@
+				final int counterPrayer = client.getBoostedSkillLevel(Skill.PRAYER);
+				final int counterHealthX  = HEALTH_LOCATION_X;
+				final int counterPrayerX = PRAYER_LOCATION_X;
+				final String counterHealthText = Integer.toString(counterHealth);
+				final String counterPrayerText = Integer.toString(counterPrayer);
+
+				// Render the bar if a player hovers over an item and draws the icon and counters so they are always on top of the bars.
+				renderHealingBar(g, offsetHealthX , offsetHealthY,
+						maxHealth, currentHealth, height,
+						foodHealValue, HEAL_COLOR, healthImage,
+						counterHealthText, counterHealthX, counter);
+
+				renderHealingBar(g, offsetPrayerX , offsetPrayerY,
+						maxPrayer, currentPrayer, height,
+						prayerHealValue, PRAYER_HEAL_COLOR, prayerImage.getScaledInstance(IMAGE_SIZE , IMAGE_SIZE , Image.SCALE_SMOOTH),
+						counterPrayerText, counterPrayerX, counter);
+			}
+		}
+		return null;
+	}
+
+	private void renderBar(Graphics2D graphics, int x, int y, int max, int current, int height, Color filled)
+	{
+		// draw background
+		graphics.setColor(BACKGROUND);
+		graphics.fillRect(x, y, WIDTH, height);
+
+		// draw bar background
+		graphics.setColor(DEPLETED);
+		graphics.fillRect(x + PADDING, y + PADDING, WIDTH - PADDING * 2, height - PADDING * 2);
+
+		// draw bar with current health or prayer points
+		int filledHeight = getBarHeight(max, current, height);
+		graphics.setColor(filled);
+		graphics.fillRect(x + PADDING, y + PADDING + (height - filledHeight), WIDTH - PADDING * 2, filledHeight - PADDING * 2);
+	}
+
+	private void renderHealingBar(Graphics2D graphics,
+		int x, int y, int max, int current, int height,
+		int heal, Color color, Image image,
+		String counterText, int counterX, Color counter)
+	{
+
+		// Start drawing the healing bar
+		int filledHeight = getBarHeight(max, heal, height);
+		int filledCurrentHeight = getBarHeight(max, current, height);
+		int overHeal;
+		graphics.setColor(color);
+
+		if (filledHeight + filledCurrentHeight > height)
+		{
+			overHeal = filledHeight + filledCurrentHeight - height;
+			filledHeight = filledHeight - overHeal + 2;
+			color = new Color(216, 255, 139, 150);
+			graphics.setColor(color);
+			graphics.fillRect(x + PADDING, y - filledCurrentHeight + (height - filledHeight) + 3 , WIDTH - PADDING * 2, filledHeight - PADDING * 2);
+		}
+		else
+		{
+			graphics.fillRect(x + PADDING, y  - 2 - filledCurrentHeight + (height - filledHeight) + 3 , WIDTH - PADDING * 2, filledHeight + 2 - PADDING * 2);
+		}
+
+		// draw counter on top of everything else
+		graphics.setColor(counter);
+		graphics.setFont(FontManager.getRunescapeSmallFont());
+
+		if (config.enableCounter() && !config.enableSkillIcon())
+		{
+			graphics.drawString(counterText, x + counterX  + PADDING, y + 20);
+		}
+		else
+		{
+			graphics.drawString(counterText, x + counterX + PADDING, y + 35);
+			graphics.drawString(counterText, x + counterX + PADDING, y + 35);
```

Comment:
why are you drawing same string twice here? Also, you should probably use TextComponent as it has shadow

Reply:
hmm weird i can't remember adding it twice, probably a copy paste mistake and didnt look back.
removed one line

Reply:
The TextComponent use still applies, try to switch to that one.

Reply:
I'm not sure i understand what you want me change here,
Use the textcomponent for allignment?

Reply:
```java
private final TextComponent textComponent = new TextComponent()
....
render...
...
setFont..
textComponent.setPosition(position);
textComponent.setColor(color);
textComponent.render(graphics);
```